server/engagement/migrations/0001_initial.py

```diff
@@ -27,7 +27,7 @@ class Migration(migrations.Migration):
             name='AttendanceSession',
             fields=[
                 ('session_id', models.AutoField(primary_key=True, serialize=False)),
-                ('key', models.CharField(max_length=10)),
```

Comment:
we've already run the model migration in prod, so youll need to create a new migration out of this

```txt
engagement
 [X] 0001_initial
```

will likely also need to nuke your local db if you already ran this, then makemigrations again for the max_length change